Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -165,6 +165,22 @@ public KeyCommand()
             }
         }
 
+        [Command(Description = "Sign a message.")]
+        public void Sign(
+            [Argument("KEY-ID", Description = "A key UUID to sign.")]
+            Guid keyId,
+            [Argument(
+                "MESSAGE",
+                Description = "A message encoded by base64 encoding to sign."
+            )]
+            string message,
+            PassphraseParameters passphrase
+        )
+        {
+            PrivateKey key = UnprotectKey(keyId, passphrase);
+            Console.WriteLine(Convert.ToBase64String(key.Sign(Convert.FromBase64String(message))));
```

Comment:
It's trivial but could you break the line? It looks quite long.

Reply:
How about printing out the raw bytes instead of base64?

Or at least hexadecimals might be better than base64, because there are [quite many variations of base64][2], which confuse those who need to decode them.

If such behavior causing [average terminal emulators to go weird][1] concerns you, we could provide an option like `-o`/`--binary-output=FILE`:

```bash
# The signature is printed out in hexadecimals:
planet key sign 00000000-0000-0000-0000-000000000000 msg.dat

# The signature is written in the "sig.dat" file (no hex, but raw binary):
planet key sign \
  --binary-output sig.dat \
  00000000-0000-0000-0000-000000000000 \
  msg.dat

# The signature is printed out in raw bytes:
planet key sign \
  --binary-output - \
  00000000-0000-0000-0000-000000000000 \
  msg.dat

# The signature is written in the "-" file:
planet key sign \
  --binary-output ./- \
  00000000-0000-0000-0000-000000000000 \
  msg.dat
```

[1]: https://unix.stackexchange.com/q/79684/9939
[2]: https://en.wikipedia.org/wiki/Base64#Variants_summary_table

Reply:
I tried to implement the feature in [4487e53](https://github.com/planetarium/libplanet/pull/1920/commits/4487e53ec05534c15f6ad5dd63fbf6400f1d4d84) commit. `--binary-output` option is optional and it prints the signature as hexadecimal string in default behaviour.